docs-kits/kits/Circularity_KIT/page_changelog.md

```diff
@@ -6,9 +6,22 @@ sidebar_position: 1
 ---
 
 ![Circulairty kit banner](/img/kit-icons/circularity-kit-icon.svg)
```

Comment:
Please correct spelling of Circularity & Kit (capitalized)

Reply:
Done